src/components/App/App.tsx

```diff
@@ -959,10 +979,15 @@ export default class App extends React.Component<null, AppState> {
     this.socket.emit('CMD:picture', url);
   };
 
-  getMediaDisplayName = (input: string) => {
+  getMediaDisplayName = (input: any) => {
```

Comment:
This should still be a string. If there's metadata to look up (e.g. youtube video), we should resolve that elsewhere.